exercises/exercise3.js

```diff
@@ -8,5 +8,145 @@ module.exports.botScripts = [
     handler: function () {
       return 'Good Evening to You!';
     },
+    //I want the bot to find the to be have a varity of moods when asked a question.
+
+    label: 'Whats your mood bot',
+    prompt: 'Next are you happy',
+    handler: function () {
+      return 'Yes i am';
+    }, //this will tell me what my name is
+    label: 'I want to make a discord bot',
+    prompt: 'What is my #{name}',
+    handler: function (params) {
+      const customer = params.name;
+      return 'Your name is ' + params.name;
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    //This will return the users status if he is online or not
+    label: 'Whats my status',
+    prompt: 'Am offline or online?',
+    handler: function () {
+      const onlineStatus = 20;
+      const offlineStatus = 75;
+      const num = 50;
+      if (onlineStatus < 50) {
+        return 'You are online!';
+      } else if (offlineStatus > 50) {
+        return 'You are offline!';
+      } else {
+        return 'You need better internet.....';
+      }
+    },
+    isCaseSensitive: false,
+    isListening: true,
+  },
+  {
+    label: 'This is a magic eight ball buddy',
+    prompt: 'My name is #{userName}, whats my destiny',
+    handler: function (params) {
+      let userName = params.userName;
+      userName
+        ? console.log('Hello,' + userName + ' !')
+        : console.log('Hello!');
+      const randomNumber = Math.floor(Math.random() * 8);
+      let eightBall = '';
+      switch (randomNumber) {
+        case 0:
+          eightBall = 'Reconsider this question';
+          break;
+        case 1:
+          eightBall = 'Here we go agian...';
+          break;
+        case 2:
+          eightball = 'Really dude? why... like why...';
+          break;
+        case 3:
+          eightBall = 'I dont get paid enough for this';
+          break;
+        case 4:
+          eightBall = 'Its not what your doing right now bud.';
+          break;
+        case 5:
+          eightBall = 'Eh, maybe you have some purpose';
+          break;
+        case 6:
+          eightBall = 'You can be a sailor';
+          break;
+        case 7:
+          eightBall = 'Just be a programmer and live happy!';
+          break;
+      }
+      console.log(eightBall);
+    },
+    isCaseSensitive: false,
+    isListening: true,
+    isreply: true,
+  },
```

Comment:
OMG I rolled out laughing on this one - still love it. The first time I got the sailor and the second i got "not what you're doing right now bud". A fantastic concept, great execution and hilarious creativity!
I like your built in if userName contingency planning - already thinking about the user! - you utilize a beautiful ternary statement here. Well done! 
Unfortunately, I have a hard time prompting that to occur, just by the nature of this hubot code, so it almost becomes unnecessary. That being said, I think it is still worth having it - if anything to demonstrate your mad skillz!